src/guides/nft-rental-marketplace/index.md

```diff
@@ -0,0 +1,1547 @@
+    gasUsed: 130387,
+    contractAddress: null,
+    logs: [ [Object] ],
+    logsBloom: '0x00000000000000000000004001000000000000000000000000000000008000001000000000000000000000000008000000010000000000000000000020040000000000000000000000000000000000000000000000040000000000000200000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000040800000000000000000000000000000000000020000000000000020000000000000000000100000000000000000000000000000000000000000000000000000000000040000000000000800000080000000000000000000000000000000000000000000000',
+    status: true,
+    effectiveGasPrice: 2500000008,
+    type: '0x2',
+    rawLogs: [ [Object], [Object] ]
+  },
+  logs: [
+    {
+      address: '0x8f7D9a7EFFAD4d0c93CFdbB8b8A7A0BC1A600baf',
+      blockHash: '0x3606594dbcec10924a73112a481fd368257b2a949b9fc1f3cddbbd786ab8f852',
+      blockNumber: 7551210,
+      logIndex: 1,
+      removed: false,
+      transactionHash: '0xc8b9ecaeccdc502ca2c334e2ecafdea17409c631622c8e0c9b26e42ce59f8325',
+      transactionIndex: 0,
+      id: 'log_0dd47305',
+      event: 'NFTRented',
+      args: [Result]
+    }
+  ]
+}
+truffle(goerlidev)> 
+```
+
+### Write `unlist.js`
+
+```javascript
+const Marketplace = artifacts.require("Marketplace");
+const NFT_CONTRACT = "[NFT_CONTRACT]";
+const TOKEN_ID = 1;
+const PRICE = 1;
+const ERC721_ABI = [
+    {
+        "inputs": [
+          {
+            "internalType": "uint256",
+            "name": "tokenId",
+            "type": "uint256"
+          }
+        ],
+        "name": "userExpires",
+        "outputs": [
+          {
+            "internalType": "uint256",
+            "name": "",
+            "type": "uint256"
+          }
+        ],
+        "stateMutability": "view",
+        "type": "function",
+        "constant": true
+    },
+    {
+        "inputs": [{"internalType": "uint256", "name": "tokenId", "type": "uint256"}],
+        "name": "ownerOf",
+        "outputs": [{"internalType": "address", "name": "owner", "type": "address"}],
+        "stateMutability": "view",
+        "type": "function"
+    }
+];
+
+const main = async (cb) => {
+    try {
+        const marketplace = await Marketplace.deployed();
+        const nftContract = new web3.eth.Contract(ERC721_ABI, NFT_CONTRACT);
+        const expires = await nftContract.methods.userExpires(TOKEN_ID).call();
+        let value = (Math.floor((expires - Date.now()/1000)/60/60/24 + 1)) * PRICE;
+        const owner = await nftContract.methods.ownerOf(TOKEN_ID).call();
+        let txn = await marketplace.unlistNFT(NFT_CONTRACT, TOKEN_ID, {from: owner, value: value});
+        console.log(txn);
+    } catch(err) {
+        console.log(err);
+    }
+    cb();
+}
+  
+module.exports = main;
+```
+
+You might not have enough eth in the owner wallet to cover the refund. If so, jump to the bonus section to write a script to fund wallets with ETH on your ganache instance!
```

Comment:
There's a bit of inconsistency in both this paragraph (and throughout other areas of the guide) when referencing ETH. Personally I'm a fan of all caps 🙃